File: Data Structures & Algorithms/concatenation-of-array/submission-1.cpp
#include <vector>

class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> type_shift(nums.size() * 2);
        for (int i = 0; i < nums.size(); ++i){
            type_shift[i] = nums[i];
            type_shift[i+nums.size()] = nums[i];
        }
        return type_shift;
    }
};

File: Data Structures & Algorithms/max-consecutive-ones/submission-1.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        counter = 0
        maxOne = 0
        for i in nums:
            if i==1:
                print(i)
                counter +=1
            else:
                maxOne = max(counter, maxOne)
                counter =0
        maxOne = max(counter, maxOne)
        return maxOne